oereb_client/static/src/component/map/map.js

```diff
@@ -109,6 +110,20 @@ const OerebMap = function () {
   }
   const [availabilityLayer] = useState(new LayerClass(availabilityConfig));
 
+  // Create mask_surrounding layer
+  const maskSurroundingConfig = {
+    preload: Infinity,
+    source: new SourceClass({
+      url: config.mask_surrounding.url
+    }),
+    opacity: 0.6,
+    zIndex: 10000
```

Comment:
Use a higher zIndex (e.g. 40000) to display mask layer on top of all other layers.